### PR TITLE
LRDOCS-5890, Elasticsearch 6.5

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-5890, updated installation steps to Elasticsearch 6.5.

### CODE REVIEW

**Comment by lipusz** [comment]
**Started reviewing...** :mag:

File: discover/deployment/articles-dxp/01-installing-search-engine/02-enterprise-search/00-xpack-intro.markdown [hunk 1]
```diff
@@ -12,81 +12,25 @@ monitoring and security. A Liferay Enterprise Search Standard subscription gets
...
+3.  Configure the connectors with the proper credentials and encryption
     information.
 
-5.  Restart Elasticsearch. These steps require a full cluster restart.
+4.  Restart Elasticsearch. These steps require a full cluster restart.
 
 Following these instructions gives you a working installation of Elasticsearch
 communicating freely with @product@. Read Elastic's documentation to learn about
 additional configuration options, features, and the architecture of
-[X-Pack](https://www.elastic.co/guide/en/elasticsearch/reference/6.1/configuring-security.html).
```
Comment (lipusz) [inline discover/deployment/articles-dxp/01-installing-search-engine/02-enterprise-search/00-xpack-intro.markdown]: Hey @rbohl 

We should keep the steps to install X-Pack for 6.1.x versions as it's optional for customers to move to ES 6.5. Either move it to a new sub-page or add it back as a sub-chapter.

File: discover/deployment/articles-dxp/01-installing-search-engine/02-enterprise-search/01-xpack-security.markdown [hunk 2]
```diff
@@ -3,26 +3,35 @@
...
 first-use passwords for the first time. To update a password subsequently, use
 Kibana's UI or the 
-[Change Password API](https://www.elastic.co/guide/en/elasticsearch/reference/6.1/security-api-change-password.html).
+[Change Password API](https://www.elastic.co/guide/en/elasticsearch/reference/6.5/security-api-change-password.html).
 
 The `interactive` argument sets the passwords for all built-in users. The
 configuration shown in these articles assumes you set all of the passwords to
 *liferay*. Of course, that's not recommended for production systems.
 
-    ./bin/x-pack/setup-passwords interactive
```
Comment (lipusz) [inline discover/deployment/articles-dxp/01-installing-search-engine/02-enterprise-search/01-xpack-security.markdown]: Keep the 6.1.x command and add note.

File: discover/deployment/articles-dxp/01-installing-search-engine/02-enterprise-search/01-xpack-security.markdown [hunk 3]
```diff
@@ -43,25 +52,24 @@ the corresponding configuration files.
...
+[Generate a node certificate](https://www.elastic.co/guide/en/elasticsearch/reference/6.5/configuring-tls.html#node-certificates)
 for each node. You can, of course, use a Certificate Authority to obtain node 
 certificates.
 
 1.  Create a certificate authority, using 
-      [X-Pack's `certutil`](https://www.elastic.co/guide/en/elasticsearch/reference/6.1/certutil.html)
+      [X-Pack's `certutil`](https://www.elastic.co/guide/en/elasticsearch/reference/6.5/certutil.html)
       command:
 
-        ./bin/x-pack/certutil ca --pem --ca-dn CN=localhost
```
Comment (lipusz) [inline discover/deployment/articles-dxp/01-installing-search-engine/02-enterprise-search/01-xpack-security.markdown]: Keep 6.1 command too.

File: discover/deployment/articles-dxp/01-installing-search-engine/02-enterprise-search/01-xpack-security.markdown [hunk 4]
```diff
@@ -43,25 +52,24 @@ the corresponding configuration files.
...
+        ./bin/elasticsearch-certutil ca --pem --ca-dn CN=localhost
 
     This generates a ZIP file. Unzip the contents somewhere safe.
 
 2. Generate X.509 certificates and private keys using the CA from Step 1. For
    example:
 
-        ./bin/x-pack/certutil cert --pem --ca-cert /path/to/ca.crt --ca-key /path/to/ca.key --dns localhost --ip 127.0.0.1 --name localhost
```
Comment (lipusz) [inline discover/deployment/articles-dxp/01-installing-search-engine/02-enterprise-search/01-xpack-security.markdown]: Keep 6.1 command or add a main note at the beginning of the section to inform readers that on 6.1.x version the path to the command is different.

File: discover/deployment/articles-dxp/01-installing-search-engine/02-enterprise-search/01-xpack-security.markdown [hunk 5]
```diff
@@ -137,7 +145,7 @@ these contents:
 Note that the `password` should match what you set during the X-Pack password
 setup above. 
 
-Enable authentication by setting authentication to `required` and providing the
```
Comment (lipusz) [inline discover/deployment/articles-dxp/01-installing-search-engine/02-enterprise-search/01-xpack-security.markdown]: Keep 6.1 property name too.

File: discover/deployment/articles-dxp/01-installing-search-engine/02-enterprise-search/02-xpack-monitoring.markdown [hunk 6]
```diff
@@ -10,23 +10,34 @@ Enterprise Search Standard subscription (included with Premium) is necessary for
...
 This document assumes you're enabling security *and* monitoring, though
 differences in the process are noted as appropriate.
 
 For the X-Pack installation procedure, refer to the 
 [X-Pack security article](/discover/deployment/-/knowledge_base-7-1/installing-x-pack-security).
 
-Start with the installation of Kibana.
+Start by enabling data collection in Elasticsearch.
+
+## Enable Data Collection [](id=enable-data-collection)
```
Comment (lipusz) [inline discover/deployment/articles-dxp/01-installing-search-engine/02-enterprise-search/02-xpack-monitoring.markdown]: If it's mandatory for 6.5.x, then add a note.

File: discover/deployment/articles-dxp/01-installing-search-engine/02-enterprise-search/02-xpack-monitoring.markdown [hunk 7]
```diff
@@ -10,23 +10,34 @@ Enterprise Search Standard subscription (included with Premium) is necessary for
...
+
+Monitoring is enabled on Elasticsearch by default, but data collection isn't.
+To use Kibana to collect data on your Elasticsearch cluster, enable data
+collection by adding this line to `elasticsearch.yml`.
+
+    xpack.monitoring.collection.enabled: true
+
+Now install Kibana.
 
 ## Install Kibana [](id=install-kibana)
```
Comment (lipusz) [inline discover/deployment/articles-dxp/01-installing-search-engine/02-enterprise-search/02-xpack-monitoring.markdown]: I'd replace the link with https://help.liferay.com/hc/en-us/articles/360016511651-Liferay-Enterprise-Search-Compatibility-Matrix.

File: discover/deployment/articles-dxp/01-installing-search-engine/02-enterprise-search/02-xpack-monitoring.markdown [hunk 8]
```diff
@@ -79,15 +86,16 @@ these steps or refer to
         ./bin/kibana
 
     and navigate to `localhost:5601`. Log in with a 
-    [user](https://www.elastic.co/guide/en/x-pack/6.1/native-realm.html#native-add)
+    [user](https://www.elastic.co/guide/en/x-pack/6.5/native-realm.html#native-add)
     who has the `kibana_user` 
-    [role](https://www.elastic.co/guide/en/x-pack/6.1/built-in-roles.html).
+    [role](https://www.elastic.co/guide/en/x-pack/6.5/built-in-roles.html) 
+    or a superuser (like the `elastic` user).
```
Comment (lipusz) [inline discover/deployment/articles-dxp/01-installing-search-engine/02-enterprise-search/02-xpack-monitoring.markdown]: I guess this is new as of 6.5, so add a note.

File: discover/deployment/articles-dxp/01-installing-search-engine/02-enterprise-search/02-xpack-monitoring.markdown [hunk 9]
```diff
@@ -141,10 +149,12 @@ the LPKG file by copying it into the `Liferay Home/deploy` folder.
     settings are picked up by your running instance. There's no need to restart
     the server.
 
-4.  There's one more setting to add to Kibana itself. It sets Kibana's base path
-    to let the Monitoring Portlet act as a proxy for Kibana's monitoring UI. Add
-    this to `kibana.yml`:
+4.  There are two more settings to add to Kibana itself. The first tells Kibana
```
Comment (lipusz) [inline discover/deployment/articles-dxp/01-installing-search-engine/02-enterprise-search/02-xpack-monitoring.markdown]: If this is new as of 6.5.x, add a note.

File: discover/deployment/articles-dxp/01-installing-search-engine/02-enterprise-search/02-xpack-monitoring.markdown [hunk 10]
```diff
@@ -163,7 +173,7 @@ the LPKG file by copying it into the `Liferay Home/deploy` folder.
     First, navigate to Elasticsearch Home and generate a PKSC#12 certificate
     from the CA you created when setting up X-Pack security:
 
-        ./bin/x-pack/certutil cert --ca-cert /path/to/ca.crt --ca-key /path/to/ca.key --ip 127.0.0.1 --dns localhost --name localhost --out /path/to/Elasticsearch_Home/config/localhost.p12
+        ./bin/elasticsearch-certutil cert --ca-cert /path/to/ca.crt --ca-key /path/to/ca.key --ip 127.0.0.1 --dns localhost --name localhost --out /path/to/Elasticsearch_Home/config/localhost.p12
```
Comment (lipusz) [inline discover/deployment/articles-dxp/01-installing-search-engine/02-enterprise-search/02-xpack-monitoring.markdown]: 6.1 vs. 6.5 note

File: discover/deployment/articles-dxp/03-configuring-liferay/03-liferay-portal-clustering/03-clustering-search.markdown [hunk 11]
```diff
@@ -19,16 +17,12 @@ other the list of all IP addresses; in case of a node going down, the proper
...
 
 Once @product@ servers have been properly configured as a cluster, deploy the
-Liferay Solr 5 Adapter on all nodes. (This app is available for download from
-Liferay Marketplace
-[here](https://web.liferay.com/marketplace/-/mp/application/78803899).) Create a
-Solr Cloud (cluster) managed by _Apache Solr Zookeeper_. Connect the @product@
-cluster to Zookeeper and finish the final configurations to connect the two
-clusters.
--->
+Liferay Connector to Solr on all nodes. (This app is available for download
```
Comment (lipusz) [inline discover/deployment/articles-dxp/03-configuring-liferay/03-liferay-portal-clustering/03-clustering-search.markdown]: Write "Liferay Connector to Solr 7"

File: discover/deployment/articles/00-deploying-liferay/10-installing-elasticsearch.markdown [hunk 12]
```diff
@@ -80,7 +77,15 @@ A JSON document is returned that looks similar to this:
     }
 
 The version of Elasticsearch that's running is the value of the `"number"` field.
-In this example, it's 6.1.3. 
+In this example, it's 6.5.1. 
+
+Shut down the @product@ server. In a local, single-machine testing environment,
```
Comment (lipusz) [inline discover/deployment/articles/00-deploying-liferay/10-installing-elasticsearch.markdown]: Can you add this in a "Note" section? Like the
>Note: the only current officially supported version is OpenAM 13. 

note in https://dev.liferay.com/discover/deployment/-/knowledge_base/7-1/opensso-single-sign-on-authentication.

File: discover/deployment/articles-dxp/01-installing-search-engine/02-enterprise-search/02-xpack-monitoring.markdown (hunk 8 above)
Comment (rbohl) [reply discover/deployment/articles-dxp/01-installing-search-engine/02-enterprise-search/02-xpack-monitoring.markdown]: This is just an enhancement, not specific to 6.5.

**Comment by rbohl** [comment]
@lipusz check out what i did in this commit. what do you think?

https://github.com/rbohl/liferay-docs/commit/aca7ad3003bb9182a04ab40612c5a2be5e77e20e

I'd like to get this in to Rich this week, otherwise it won't get published until after the New Year.

**Comment by rbohl** [comment]
@lipusz check out what i did in this commit. what do you think?

https://github.com/rbohl/liferay-docs/commit/aca7ad3003bb9182a04ab40612c5a2be5e77e20e

I'd like to get this in to Rich this week, otherwise it won't get published until after the New Year.

**Comment by lipusz** [comment]
Did you comment on this just now?

**Comment by rbohl** [comment]
Yeah, but I accidentally sent a duplicate of yesterday's  comment. So I deleted it.

**Comment by lipusz** [comment]
Ah, ok. :)

**Comment by lipusz** [comment]
Hey @gergelyszaz: can you please update OriginAl to handle the 7.1.x branch too from the liferay-docs repo? Thanks!

**Comment by gergelyszaz** [comment]
Hi @lipusz , sure!

**Pull request closed without merging**